addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/automation.html

```diff
@@ -68,6 +68,14 @@ <H2>Jobs</H2>
 <li><a href="job-requestor.html">requestor</a> - crafts specific requests to send to the corresponding targets</li>
 <li><a href="job-ascan.html">activeScan</a> - runs the active scanner</li>
 </ul>
+
+<H3>Importance of Job Order</H3>
+The order of jobs is relevant and important. For example: 
```

Comment:
Say also that the jobs are executed in the order they are defined in the plan? Maybe this should be said beforehand when explaining what a plan is and jobs are.